python/storage-cleaner/src/appwrite.py

```
@@ -0,0 +1,65 @@
+                raise RuntimeError(
+                    f"Failed to list files from bucket {bucket_id}: {str(e)}"
+                ) from e
+            files = response.get("files", [])
+
+            for f in files:
+                try:
+                    file_id = f.get("$id")
+                    if file_id:
+                        self.storage.delete_file(bucket_id, file_id)
+                        deleted_files_count += 1
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)